core/src/main/java/org/infinispan/context/EntryLookup.java

```diff
@@ -38,4 +39,9 @@
 
    void removeLookedUpEntry(Object key);
 
+   default void removeLookedUpEntries(Collection<?> keys) {
```

Comment:
No; I think the default implementation is good enough. You wouldn't put anything else to impls.